core/gather/session.js

```diff
@@ -94,7 +94,13 @@ class ProtocolSession extends CrdpEventEmitter {
       }));
     });
 
-    const resultPromise = this._cdpSession.send(method, ...params);
+    const resultPromise = this._cdpSession.send(method, ...params).catch(err => {
+      // We set up our own protocol timeout system, so we should ignore protocol timeouts emitted by puppeteer.
+      // https://github.com/GoogleChrome/lighthouse/issues/15510
+      if (/'protocolTimeout'/.test(err)) return;
```

Comment:
Won't this resolve with `undefined`, though, if `timeoutPromise` doesn't win the race?

Reply:
Good point. I guess if puppeteer gives us a protocol timeout error that means they are done processing the command. Even if we wait around longer, we won't get a result.

Reply:
I suppose an better solution would be to just ignore this type of error for our `Page.navigate` call specifically? IDK but this clearly needs more work.